lib/msal-browser/src/cache/DatabaseStorage.ts

```diff
@@ -71,20 +88,26 @@ export class DatabaseStorage<T> implements IAsyncStorage<T> {
      */
     async getItem(key: string): Promise<T | null> {
         await this.validateDbIsOpen();
-
         return new Promise<T>((resolve, reject) => {
             // TODO: Add timeouts?
-            if (!this.db) {
+            const db = this.db;
```

Comment:
Can you please explain this?

Reply:
I was also curious as to why this was needed everywhere.

Reply:
We should address the race rather than attempting to avoid it this way

Reply:
I undid the changes related to the constant assignment and I haven't been able to repro the race that was happening. It seems something else must have been off when I was testing before.